src/Warehouse/Application/Background/ProductsSyncService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Application.Configuration;
+using Application.Features.Products.Synchronize;
+using MediatR;
+using Microsoft.Extensions.Options;
+using Quartz;
+
+namespace Application.Background
+{
+    public class ProductsSyncService : IJob
+    {
+        private readonly IMediator _mediator;
+
+        private readonly ProductsSourceConfig _productsSourceConfig;
+
+        public ProductsSyncService(
+            IMediator mediator,
+            IOptions<ProductsSourceConfig> productsSourcs)
+        {
+            _mediator = mediator;
+            _productsSourceConfig = productsSourcs.Value;
+        }
+
+        public async Task Execute(IJobExecutionContext context)
+        {
+            try
+            {
+                await _mediator.Send(new SyncProductsCommand(_productsSourceConfig.Uri));
+            }
+            catch { }
```

Comment:
To be honest I do not log here, because I want (and I do) rely on the Logging pipeline behavior. I have a common approach for all the handlers - if there is a error inside to log the error into the logs.

What do you think?